scripts/tests/test_sqa_check.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
```

Comment:
Currently this won't get run because we don't do a `run_tests` in `scripts`. I will get this branch and try it out